compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -1059,6 +1059,7 @@ class Definitions {
   @tu lazy val RetainsByNameAnnot: ClassSymbol = requiredClass("scala.annotation.retainsByName")
 
   @tu lazy val JavaRepeatableAnnot: ClassSymbol = requiredClass("java.lang.annotation.Repeatable")
+  @tu lazy val JavaAnnotationAnnot: ClassSymbol = requiredClass("java.lang.annotation.Annotation")
```

Comment:
`java.lang.annotation.Annotation` is not itself an annotation. So it should be called `JavaAnnotationClass` instead of `JavaAnnotationAnnot`.

Arguably it should also be located near `AnnotationClass` under `// Annotation base classes`.

Reply:
thanks! implemented in https://github.com/lampepfl/dotty/pull/19074